src/components/cesium_map/api/spatial.js

```diff
@@ -423,8 +426,20 @@ export class ISamplesSpatial {
       },
     });
 
+    // entity for infoBox
+    this.selectedPoints = this.viewer.entities.add({
+      point: {
+        show: false
+      }
+    })
+
     // record the last interactive point primitive
     this.pointprimitive = null;
+
+    // we need to enable allow-scripts to open link in the iframe
+    // but this might not be a safe way if we don't trust the link source
+    this.viewer.infoBox.frame.setAttribute('sandbox', 'allow-same-origin allow-scripts allow-popups allow-forms');
+    this.viewer.infoBox.frame.removeAttribute("src");
```

Comment:
We still need to enable `allow-scripts` to popup a new window in the `iframe`. And we need to remove `src` to prevent a new tab to use the same URL domain.